build.gradle

```diff
@@ -0,0 +1,34 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+	repositories {
+		google()
+		mavenCentral()
+		maven {
+			url 'https://jitpack.io'
+		}
+	}
+}
+plugins {
+	id 'com.android.application' version '7.2.0' apply false
+	id 'com.android.library' version '7.2.0' apply false
+	id 'org.jetbrains.kotlin.android' version '1.7.0' apply false
+	id 'org.jetbrains.kotlin.jvm' version '1.7.0'
+	id 'com.diffplug.spotless' version '6.8.0' apply true
+}
+
+//allprojects {
```

Comment:
Do we need this?

Reply:
Yeah, I mentioned it earlier. I want to configure it on the highest level, then link it to the modules and include it in the build flow. And it's WIP.